src/components/message/Message.styles.ts

```diff
@@ -0,0 +1,46 @@
+import { chatShadow } from '~/styles/app-theme/custom-shadows'
+
+export const styles = {
+  userInfoStyles: {
+    root: {
+      flexDirection: 'row',
+      gap: '10px'
+    },
+    avatar: {
+      width: '40px',
+      height: '40px',
+      borderRadius: '5px'
+    },
+    info: {
+      flexDirection: 'row',
+      alignItems: 'baseline',
+      gap: '10px'
+    },
+    name: {
+      typography: 'caption',
```

Comment:
you are using the same styles below, you can move it to an object and destructure where needed or use the same property for two elements

Reply:
fixed it